docs/pkp/wallets/sendingTxs.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 3
+---
+
+# Sending Transactions with PKPs
+
+:::note
+**STATE OF THE NETWORK**
+
+PKPs are still heavily in development and should NOT BE USED to transact live assets that hold monetary value. DO NOT store assets you are not prepared to lose. During this period, we're grateful for feedback on how to improve the docs and examples.
+:::
+
+You can use the [Lit PKP SDK](https://github.com/LIT-Protocol/lit-pkp-sdk) (an extension of the PKPWallet package) to handle Ethereum JSON RPC requests for signing messages, data, and transactions and sending transactions with PKPs.
```

Comment:
The `lit-pkp-sdk` is going to be moved to the new `js-sdk`. Maybe we can add a note like -

"Note: This will be migrated to the new Typescript SDK. The package import and object initialization may change, but the core functionality and methods should remain the same."